run_nohup.sh

```
@@ -0,0 +1,67 @@
+    echo "Examples:"
+    echo "  $0 \"Create a game concept\" --evolve --generations 3 --population 10"
+    echo "  $0 \"What is consciousness\" --evolve"
+    echo "  $0 \"Analyze this problem\" --temperature 1.5"
+    echo ""
+    echo "This script runs mad_spark_alt in the background to avoid 2-minute timeout issues."
+    exit 1
+fi
+
+# Ensure we're in the right directory
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)